packages/gatsby/lib/schema/index.js

```diff
@@ -27,13 +28,13 @@ async function buildSchema() {
     }),
   })
 
+  await writeSchema(schema)
+
   console.timeEnd(`building schema`)
   store.dispatch({
     type: `SET_SCHEMA`,
     payload: schema,
```

Comment:
maybe we don't need to write out hte schema. perhaps just pass it to the compiler from redux (just now noticing this)